### DISCUSSION TITLE
run on k3d

### DISCUSSION THREAD

**Discussion by fragolinux** [gravitational/teleport#27586] [Q&A]
hi, just testing teleport 13 on a k3d 5.5.1 cluster running on a vps (dns records are already ok)...
if i just run:
`k3d cluster create`
the pod will remain pending because ports are already opened by k3d load balancer...

if i run as such, port 443 never gets opened on my system, so no access at all...

```
k3d cluster create teleport --k3s-arg "--disable=traefik@server:*"

cat << EOF > teleport-cluster-values.yaml
clusterName: my.domain
proxyListenerMode: multiplex
acme: true
acmeEmail: my@email
EOF

helm install teleport-cluster teleport/teleport-cluster \
  --create-namespace \
  --namespace=teleport-cluster \
  --version 13.1.0 \
  --values teleport-cluster-values.yaml

```

**Comment by webvictim** [comment]
Change `service.type` to something that isn't a `LoadBalancer`?

```yaml
service:
  type: ClusterIP
```

**Reply by webvictim** [reply]
You could also try using an `Ingress` as per this PR: https://github.com/gravitational/teleport/pull/25815

**Comment by fragolinux** [comment]
I'll wait for some example, if someone could help, thanks
